examples/standalone/02_hello_window/src/main.rs

```diff
@@ -0,0 +1,169 @@
+use std::sync::Arc;
+
+use winit::application::ApplicationHandler;
+use winit::event::WindowEvent;
+use winit::event_loop::{ActiveEventLoop, ControlFlow, EventLoop};
+use winit::window::{Window, WindowId};
+
+struct State {
+    surface: wgpu::Surface<'static>,
+    device: wgpu::Device,
+    adapter: wgpu::Adapter,
+    queue: wgpu::Queue,
+    size: winit::dpi::PhysicalSize<u32>,
+}
+
+impl State {
+    // Creating some of the wgpu types requires async code
+    async fn new(window: Arc<Window>) -> State {
+        let size = window.inner_size();
+
+        // The instance is a handle to our GPU
+        // Backends::all => Vulkan + Metal + DX12 + Browser WebGPU
+        let instance = wgpu::Instance::new(&wgpu::InstanceDescriptor {
+            backends: wgpu::Backends::PRIMARY,
+            ..Default::default()
+        });
+
+        let surface = instance.create_surface(Arc::clone(&window)).unwrap();
+
+        let adapter = instance
+            .request_adapter(&wgpu::RequestAdapterOptions {
+                power_preference: wgpu::PowerPreference::default(),
+                compatible_surface: Some(&surface),
+                force_fallback_adapter: false,
```

Comment:
Maybe use `..Default::default()` here too, instead of mentioning the relatively esoteric `force_fallback_adapter` option? That will also make the code usable with more future versions of `wgpu` that might introduce more options.